.gitmodules

```diff
@@ -4,7 +4,7 @@
 
 [submodule "submodules/dccrg"]
 	path = submodules/dccrg
-	url = https://github.com/fmihpc/dccrg.git
+	url = https://github.com/ursg/dccrg.git
```

Comment:
Shouldn't the relevant commit be visible via the PR in fmihpc - no need to point to ursg/dccrg?

Reply:
Looks like that works indeed. Nice!